frontend/next.config.js

```diff
@@ -7,6 +7,8 @@ const domain = process.env.NOTIFY_DOMAIN_NAME ?? 'localhost:3000';
 
 const nextConfig = (phase) => {
   const isDevServer = phase === PHASE_DEVELOPMENT_SERVER;
+  const includeAuthPages =
```

Comment:
I've added this in because the automated tests running in CI do a production build of the we application.

This build does not include the the auth pages. So we're unable to login to the app. 

I'm not entirely happy with it. Any alternative suggestions would be great. 

--

In the automated tests we take what is effectively a snapshot of the cookies and store them in a file called `user.json`. This information is then passed into the browser context on each test run.

Reply:
Lets have a chat with the team about this, see if we can find a more elegant solution.